README.md

```diff
@@ -675,3 +714,37 @@ Your changes will only take effect in new terminal windows.
 [running_gif]: doc/images/opensim_running.gif
 [simple_example_gif]: doc/images/opensim_double_pendulum_muscle.gif
 [java]: http://www.oracle.com/technetwork/java/javasebusiness/downloads/java-archive-downloads-javase6-419409.html
+
+#### For the impatient (Ubuntu)
+
+In **Terminal** --
+```shell
+ sudo add-apt-repository ppa:george-edison55/cmake-3.x
+ sudo apt-add-repository ppa:fenics-packages/fenics-exp
+ sudo apt-get update
+ sudo apt-get install git cmake cmake-curses-gui clang-3.6 \
+                      freeglut3-dev libxi-dev libxmu-dev \
+                      liblapack-dev swig3.0 python-dev openjdk-7-jdk
+ sudo rm -f /usr/bin/cc /usr/bin/c++
+ sudo ln -s /usr/bin/clang-3.6 /usr/bin/cc
+ sudo ln -s /usr/bin/clang++-3.6 /usr/bin/c++
+ export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
```

Comment:
You might also be able to set a CMake variable (`-DJAVA_HOME`), based on the documentation here: https://cmake.org/cmake/help/v3.0/module/FindJava.html.